examples/bare/src/root.tsx

```diff
@@ -1,12 +1,14 @@
 // @refresh reload
 import { Suspense } from "solid-js";
 import { Body, ErrorBoundary, FileRoutes, Head, Html, Meta, Routes, Scripts } from "solid-start";
+import { Title } from "@solidjs/meta";
```

Comment:
Is this the right place to import from? 
Wonder why its not clubbed into `solid-start/root`